pkg/console/subresource/deployment/deployment.go

```diff
@@ -631,7 +631,10 @@ class Thread(threading.Thread):
 		self.start()
 
 	def run(self):
-		httpd = http.server.HTTPServer(addr, http.server.SimpleHTTPRequestHandler, False)
+		server = http.server.SimpleHTTPRequestHandler
+		server.server_version = ""
```

Comment:
nit: You might just make this something like "OpenShift Downloads Server" (without any specific version). This doesn't reveal any information about the server runtime, but avoids serving an empty response header.